mne/time_frequency/tfr.py

```diff
@@ -66,6 +66,11 @@ def morlet(sfreq, freqs, n_cycles=7.0, sigma=None, zero_mean=False):
     Ws = list()
     n_cycles = np.atleast_1d(n_cycles)
 
+    freqs = np.array(freqs)
+    if np.any(freqs <= 0):
+        raise ValueError("all computed frequencies in freqs must be "
```

Comment:
What do you mean by "computed frequencies"? How about:

```
All frequencies in "freqs" must be greater than 0.
```

Reply:
Done!